index.d.ts

```diff
@@ -30,35 +27,62 @@ declare namespace ReactMarkdown {
     [key: string]: string | Renderer<any>
   }
 
-  interface ReactMarkdownProps {
+  interface ReactMarkdownPropsBase {
     readonly className?: string
-    readonly source?: string
     readonly sourcePos?: boolean
     readonly includeNodeIndex?: boolean
     readonly rawSourcePos?: boolean
-    readonly escapeHtml?: boolean
-    readonly skipHtml?: boolean
     readonly allowNode?: (node: mdast.Content, index: number, parent: NodeType) => boolean
-    readonly allowedTypes?: NodeType[]
-    readonly disallowedTypes?: NodeType[]
     readonly linkTarget?: string | LinkTargetResolver
     readonly transformLinkUri?:
       | ((uri: string, children?: ReactNode, title?: string) => string)
       | null
     readonly transformImageUri?:
       | ((uri: string, children?: ReactNode, title?: string, alt?: string) => string)
       | null
-    readonly unwrapDisallowed?: boolean
     readonly renderers?: {[nodeType: string]: ElementType}
+    /**
+     * @deprecated please use plugins
```

Comment:
`astPlugins` was never documented in the props, but it is still recommended by the readme in the HTML section as that’s the only way to have HTML in Markdown now.

There are a lot of bugs with it though.

Also: I soft deprecated `escapeHtml` in favor of `allowDangerousHtml`

Reply:
resolved in 2872089